src/secrets/CommandMkdir.ts

```diff
@@ -50,16 +56,57 @@ class CommandMkdir extends CommandPolykey {
           },
           logger: this.logger.getChild(PolykeyClient.name),
         });
-        await binUtils.retryAuthentication(
-          (auth) =>
-            pkClient.rpcClient.methods.vaultsSecretsMkdir({
-              metadata: auth,
-              nameOrId: secretPath[0],
-              dirName: secretPath[1],
-              recursive: options.recursive,
-            }),
-          meta,
+        const response = await binUtils.retryAuthentication(async (auth) => {
+          const response =
+            await pkClient.rpcClient.methods.vaultsSecretsMkdir();
+          const writer = response.writable.getWriter();
+          let first = true;
+          for (const [vault, path] of secretPaths) {
+            await writer.write({
+              nameOrId: vault,
+              dirName: path,
+              metadata: first
+                ? { ...auth, options: { recursive: options.recursive } }
+                : undefined,
+            });
+          }
+          await writer.close();
+          return response;
+        }, meta);
+
+        // As git does not track empty directories in version control, we need
+        // to inform the user about this.
+        process.stderr.write(
+          'WARNING: Empty directories will not be affected by modifications to the vault state (version change, sharing/cloning, etc.)\n',
         );
```

Comment:
This is not how we should warn people about this behaviour. This information should be part of the command description and documentation.

Reply:
Okay, so I just moved this warning into the command description itself.

```ts
    this.description(
      'Create a Directory within a Vault. Empty directories are not a part of the vault and will not be shared when cloning a Vault.',
    );
```